frontend/src/metabase/visualizations/components/ChartSettings/BaseChartSettings/BaseChartSettings.tsx

```diff
@@ -75,7 +75,7 @@ export const BaseChartSettings = ({
     [chartSettings, series],
   );
 
-  const styleWidget = useMemo(() => {
+  const styleWidget: Widget | null = useMemo(() => {
```

Comment:
Nit: I think this will give slightly easier to read errors when a mistake is made:

```suggestion
  const styleWidget = useMemo((): Widget | null => {
```